app/src/main/java/com/android/sample/model/fridge/localData/FridgeItemDao.kt

```diff
@@ -0,0 +1,18 @@
+package com.android.sample.model.fridge.localData
+
+import androidx.room.Dao
+import androidx.room.Delete
+import androidx.room.Insert
+import androidx.room.OnConflictStrategy
+import androidx.room.Query
+import com.android.sample.resources.C.Tag.FRIDGE_DB_COL_NAME_TABLE
+
+@Dao
+interface FridgeItemDao {
+
+  @Insert(onConflict = OnConflictStrategy.REPLACE) suspend fun insert(fridgeItem: FridgeItemEntity)
+
+  @Delete suspend fun delete(fridgeItem: FridgeItemEntity)
+
+  @Query("SELECT * FROM $FRIDGE_DB_COL_NAME_TABLE") suspend fun getAll(): List<FridgeItemEntity>
```

Comment:
Yes, exactly! Please note that the `@Insert(onConflict = OnConflictStrategy.REPLACE)` annotation for the `suspend fun insert(fridgeItem: FridgeItemEntity)` method replaces an item in case of a conflict. This approach is also utilized as a method to update the database.

Reply:
Thanks for clarifying!